docs/evaluate/major-components.mdx

```diff
@@ -19,15 +19,12 @@ Temporal has four major components:
 
 - **Temporal SDK** used to build applications and run applications.
 - **Temporal Service** that maintains the state of the applications.
-- **Temporal CLI** used to interact with the Temporal Service for a variety of purposes.
-- **Temporal UI** primary used to view the state of applications.
+- **Temporal CLI** used to interact with the Temporal Service.
+- **Temporal UI** primarily used to view the state of applications.
 
 ## Temporal SDK
 
-Temporal SDKs (software development kits) are an open source collection of tools, libraries, and APIs that enable Temporal Application development.
-
-They offer a [Temporal Client](/encyclopedia/temporal-sdks#temporal-client) to interact with a [Temporal Service](#temporal-service), APIs to develop your [Temporal Application](/encyclopedia/temporal-sdks#temporal-application), and APIs to run horizontally scalable [Workers](/workers#worker).
-
+Temporal SDKs (software development kits) are an open source collection of tools, libraries, and APIs.
```

Comment:
I thought it was more like: A Temporal SDK is a language-specific library.  It provides APIs that enable developers to interact with a Temporal Service, to develop Workflow Definitions and the Workers and client applications that initiate and manage them.

Reply:
If you look at what was there before, this is exactly what we had, but I agree with @jsundai in removing it.

We want high-level almost marketing-link info here.